pkg/chaincode/lifecycle.go

```diff
@@ -6,5 +6,17 @@ SPDX-License-Identifier: Apache-2.0
 package chaincode
 
 const (
-	lifecycleChaincodeName = "_lifecycle"
+	lifecycleChaincodeName        = "_lifecycle"
+	approveTransactionName        = "ApproveChaincodeDefinitionForMyOrg"
+	commitFuncName                = "CommitChaincodeDefinition"
+	queryInstalledTransactionName = "QueryInstalledChaincodes"
+	checkCommitReadinessFuncName  = "CheckCommitReadiness"
```

Comment:
It would be nice to rename this to `checkCommitReadinessTransactionName` for consistency